cmd/dmsg-discovery/internal/api/api.go

```diff
@@ -97,7 +97,7 @@ func (a *API) getEntry() func(w http.ResponseWriter, r *http.Request) {
 // Method: POST
 // Args:
 //	json serialized entry object
-func (a *API) setEntry() func(w http.ResponseWriter, r *http.Request) {
+func (a *API) setEntry(enableLoadTesting bool) func(w http.ResponseWriter, r *http.Request) {
```

Comment:
What do you think about storing the value of `enableLoadTesting` in `API` instead of passing to methods?